eng/pipelines/apiview-review-gen-cadl.yml

```diff
@@ -0,0 +1,87 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+variables:
+  NodeVersion: '16.x'
+
+jobs:
+- job: CreateSwaggerReviewCodeFile
+  displayName: 'Create Swagger API review token file'
+
+  variables:
+  - template: /eng/pipelines/templates/variables/globals.yml
+
+  steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: $(NodeVersion)
+    displayName: "Use Node $(NodeVersion)"
+
+  - pwsh: |
+      npm install -g cadl
+      npm install -g @cadl-lang/compiler
+    displayName: "Install cadl compiler"
+
+  - pwsh: |
+      Write-Host "Review Details Json: $(Reviews)"

```

Comment:
Where is this Reviews variable coming from? Should it be a runtime parameter to this pipeline? What happens if this is not set or set incorrectly?

Reply:
Also based on the (s) at the end I'd assume you could give multiple reviews but this step seems to only handle a single review.

Reply:
Yes. I would like to discuss this with you. I am processing only 1 review to utilize sparse checkout and solution involves changing sandboxing framework for all languages.

Reply:
Fixed to support multiple reviews